src/math/CMakeLists.txt

```diff
@@ -6,6 +6,10 @@ endif()
 
 add_local_sources(sof numbers.c trig.c decibels.c iir_df2t_generic.c iir_df2t_hifi3.c)
 
+if(CONFIG_MATH_ARITHMETIC)
+	 add_local_sources(base2log.c power.c)
```

Comment:
This should be "add_local_sources(sof base2log.c power.c)". It won't build like it is.